e2e/packages/contracts/worlds.json

```diff
@@ -1,5 +1,5 @@
 {
   "31337": {
-    "address": "0x5FbDB2315678afecb367f032d93F642f64180aa3"
+    "address": "0xA7c59f010700930003b33aB25a7a0679C860f29c"
```

Comment:
wonder if this will muck with e2e tests or if this will get overwritten in CI after deploy